katalog/curator-s3/README.md

```diff
@@ -1,32 +1,37 @@
 # Curator S3
 
+<!-- <KFD-DOCS> -->
+
 Curator helps you manage your Elasticsearch indices and snapshots via various
 operations like delete, snapshot, and shard allocation routing. It's mainly used
 to managed retention of your infrastructure logs to a given value.
 
+For `Curator` to work with `s3`, we will require an `AWS s3` compatible object
```

Comment:
is s3 the right spelling or should it be S3?

Reply:
fixed

Reply:
fixed it!